src/app/content/components/Page/contentDOMTransformations.ts

```diff
@@ -117,3 +119,9 @@ function wrapSolutions(rootEl: HTMLElement, intl: IntlShape) {
     `;
   });
 }
+
+function prefixResources(rootEl: HTMLElement) {
+  rootEl.querySelectorAll<HTMLImageElement>('img[src^="/resources/"]').forEach(
+    (el) => el.src = REACT_APP_ARCHIVE_URL + el.getAttribute('src')
+  );
+}
```

Comment:
Can only images be resources? If not, maybe the function's name could reflect that it only picks images?

Reply:
i think other things can load from resources, but i couldn't find any while i was working on this